src/routes/my/_Empty.svelte

```diff
@@ -24,7 +26,12 @@
     max-width: 100%;
   }
   :global(.night-mode) div {
-    background-image: url('/empty-page_dark.svg');
+    height: 227px;
+    background: url('/empty-page_dark.svg') no-repeat 50% 0;
```

Comment:
Other background rules are defined in a `div` selector. For night-mode we just have to change `background-image`